Update MainMenu.cpp

dylan-sacco opened the pull request:
Updates Main Menu for better comprehensiveness. this fixes the first half of git Issue #122 I would still like the see more clear instructions for the app like how to open the menu in game. 

WARNING CHANGES ARE UNTESTED.
upon request i can download and run this version on my device and analyze the changes. also i can add the last half of that issue.

dylan-sacco committed:
Update MainMenu.cpp

Updates Main Menu for better comprehensiveness

eku commented:
> WARNING CHANGES ARE UNTESTED.

Text strings are twice as long and don't fit on screen.

dylan-sacco commented:
Wondering if i can Move them up a bit, and drop them down on a new line i will see what i can do

dylan-sacco commented:
> > WARNING CHANGES ARE UNTESTED.
> 
> Text strings are twice as long and don't fit on screen.

Updated text to fit on screen, also Added a hint on how to open mod menu

dylan-sacco commented:
How do I change the version number? Other than that, it's ready to be merged.

dylan-sacco commented:
<img width="463" height="492" alt="Screenshot 2025-08-01 082048" src="https://github.com/user-attachments/assets/b5e3769d-1e4f-4ab6-9e01-2f93815176fa" />

dylan-sacco committed:
Commits on reall account
